src/pages/index.js

```diff
@@ -54,20 +87,17 @@ if (typeof window !== 'undefined') {
             return `<div class="${c}"><a href="#${link}">${item.value}</a></div>`;
         }).join("");
 
-        console.log(markup);
         tocDiv.innerHTML = '<h1 className="table-heading">Table of Contents</h1>' + markup;
     }
 
     var toc = {}
```

Comment:
I think we don't use toc anymore. It's my bad that I didn't remove it while I was working on it! Can you please remove toc and may be rename toc2 to toc?

Reply:
Besides this, everything else looks good!